homeassistant/components/notify/slack.py

```diff
@@ -68,16 +68,26 @@ def send_message(self, message="", **kwargs):
         """Send a message to a user."""
         import slacker
 
-        channel = kwargs.get('target') or self._default_channel
+        targets = kwargs.get(ATTR_TARGET, self._default_channel)
+
+        if not targets:
```

Comment:
This will not happen, you're swapping it out for `self._default_channel` (which should probably be wrapped in a list to have the same data structure.)